File: Backend/AirlineAPI/Models/FlightReservation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace AirlineAPI.Models
{
    public class FlightReservation
    {
        [Key]
        public int ReservationID { get; set; }
        [Required]
        public UserDetail ReservedUser { get; set; }
        [Required]
        public Seat ReservedSeat { get; set; }
        [Required]
        public FlightInfo2 ReservedFlight { get; set; }

    }
}

File: clover/src/app/entities/FlightReservation/flight-reservation.ts
import { User } from '../User/user';
import { Seat } from '../Seat/seat';
import { FlightInfo } from '../flightInfo/flight-info';

export class FlightReservation {
    reservationID:number;
    reservedUser:User;
    reservedSeat:Seat;
    reservedFlight:FlightInfo;

    constructor(){
        
    }
}
